requirements.txt

```diff
@@ -0,0 +1,5 @@
+Django>=2.1.3,<2.2.0
```

Comment:
Better use fixed versions in production to make you deployed code deterministic. Only libraries should have version ranges :wink:

Reply:
Updated to use fixed version of Django.